src/test/java/com/everis/boundary/CarResourcesTest.java

```diff
@@ -0,0 +1,109 @@
+package com.everis.boundary;
+
+import static org.junit.Assert.*;
+
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+
+public class CarResourcesTest {
+
+	@InjectMocks
+	private CarResources carResource;
+
+	@Mock
+	private CarService carService;
+
+	private Car car;
+
+	
+
+	@Before
+	public void setUp() throws Exception {
+		car = new Car("bmw",new Date(),"germany");
+		car.setId("e72fd0a4-f7a5-42d4-908e-7bc1dc62f857");
+	}
+
+	@Test
+	public void testCreateCar() {
+		try {
+			doNothing().when(this.carService).createCar(car);
+			Response responseTest = this.carResource.createCar(car);
+			assertEquals(Status.CREATED.getStatusCode(), responseTest.getStatus());
+			assertEquals(car, responseTest.getEntity());
+			
+		
+		} catch (Exception e) {
+			fail("create car test failed");
+		}
```

Comment:
It is not needed to catch the exception and call the junit fail method. If an exception happens the test will fail automatically.